### ISSUE TITLE
Find a better license match, rather than the ad-hoc "MIT+":

### ISSUE THREAD

**Issue by jwdonahue** [VersionSchema/VersionMeta#5]
Current license:

License
This document is licensed, as-per the LICENSE file, as derived from the MIT License. The original condition:

"The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software"

was changed to:

"The above copyright notice, a clear indication whether the content has been modified, and this permission notice shall be included in all copies or substantial portions of the Software".

In other words, MIT + tell everyone whether you simply copied the content or modified it.
